corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -12,14 +12,21 @@
 import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
```

Comment:
Please add a proper git message that explains these changes.

Reply:
Updated the comment with an explanation.  Let me know what you think.

Reply:
Github comment page is not reflected in the git commit message. Please add those details to the git commit message.

Reply:
done